src/globals/objects/_main-wrapper.scss

```diff
@@ -1,22 +1,38 @@
-// Example usage:
+// Example usage with Breadcrumbs, phase banners, back links:
 // <div class="govuk-o-width-container">
 //   <!-- Breadcrumbs, phase banners, back links are placed in here. -->
 //   <div class="govuk-o-main-wrapper">
 //       <!-- Wrapper for the main content of your page which applies padding
 //            to the top / bottom -->
 //   </div>
 // </div>
+//
+// Example usage without Breadcrumbs, phase banners, back links:
+// <div class="govuk-o-width-container">
+//   <div class="govuk-o-main-wrapper--l">
```

Comment:
Modifiers (`--`) should always be used _with_ the classes that they are modifying – in which case this would be `class="govuk-o-main-wrapper govuk-o-main-wrapper--l"`.